kotlin-checks-test-sources/src/main/kotlin/checks/EqualsOverridenWithHashCodeCheckSample.kt

```diff
@@ -61,6 +61,27 @@ class EqualsOverridenWithHashCodeCheckSample {
         }
     }
 
+
+}
+
+abstract class AmbiguousParent {
+    abstract override fun equals(other: Any?): Boolean
+}
```

Comment:
One easy test case to add in the interest of branch coverage:
```suggestion
abstract class AmbiguousParent {
    abstract override fun equals(other: Any?): Boolean
}

abstract class AmbiguousParent4 {
    abstract override fun hashCode(): Int
}
```